contracts/Vault1.sol

```diff
@@ -1,6 +1,25 @@
 //SPDX-License-Identifier: Unlicense
 pragma solidity ^0.8.4;
 
+import "@openzeppelin/contracts/utils/math/SafeMath.sol";
+
 contract Vault1 {
-    
+    // Extensions
+    using SafeMath for uint256;
```

Comment:
SafeMath is no longer required in 0.8.0 and above as its already built in (With the checked and unchecked arithmetic). Check this - https://blog.soliditylang.org/2020/12/16/solidity-v0.8.0-release-announcement/